app/main.py

```diff
@@ -1,3 +1,74 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from os import path
+
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    _path = path.join("app", "config.json")
+    with open(_path, "r") as file:
+        data_from_file = json.load(file)
+        fuel_price = data_from_file["FUEL_PRICE"]
```

Comment:
The code assumes that the JSON file contains the key 'FUEL_PRICE'. It would be safer to add error handling to manage cases where this key might be missing, such as using `data_from_file.get('FUEL_PRICE')` with a default value or raising an informative exception.